conans/test/assets/genconanfile.py

```diff
@@ -38,8 +38,13 @@ def __init__(self, name=None, version=None):
         self._package_info = {}
         self._package_id_lines = []
         self._test_lines = []
+        self._layout_text = None
         self._short_path = None
-        self._exports_sources = []
+        self._exports_sources = None
```

Comment:
Making ``exports_sources=None`` will break things, why changing it?

Reply:
Mistake in merge.